src/Module/Item/Compose.php

```diff
@@ -135,7 +135,7 @@ protected function content(array $request = []): string
 
 		$contact = Contact::getById($a->getContactId());
 
-		if ($this->config->get(local_user(), 'system', 'set_creation_date')) {
+		if (DI::config()->get(local_user(), 'system', 'set_creation_date')) {
```

Comment:
Please overwrite the `__construct()`or instead and include the Config interface instead
@MrPetovan - tbh I'm doing hard remembering the right interface names, before the restructure, it was easy - `IConfig`, now I do have to think if it was `ICanConfig` or `IHandleConfigs`, ... Maybe we should discuss about it

Reply:
`DI` was used before in this file, so I think that this is a quick solution for an error.

Reply:
Ok, but please add a follow-up removing the `DI::` calls..

Reply:
I have to have a look at where to copy it from and then have to implement it without crashing the system :-)

Reply:
@nupplaphil I'm sorry about the inconvenience, I still believe that more descriptive names are better, and we can keep discussing it.